wbsv/main.py

```diff
@@ -113,21 +115,25 @@ def usual(args):
             print("[{:02d}/{}]: <FAIL> {}".format(ind, len_links, link))
             fail += 1
 
-    print("[+]FIN!: {}".format(args.url))
+    print("[+]FIN!: {}".format(urls))
     print("[+]ALL: {}, NOW: {}, PAST: {}, FAIL: {}".format(len_links, now, past, fail))
 
 
-def cache_or_now(ind, len_links, archived_link, cached_flag):
+def wbsv_from_parser_args(args: argparse.Namespace) -> None:
+    wbsv(args.url, args.own, args.only_target, args.level, args.retry)
+
+
+def cache_or_now(ind: int, len_links: int, archived_link: str, cached_flag: bool) -> Tuple[int, int]:
     if cached_flag:
         print("[{:02d}/{}]: <PAST> {}".format(ind, len_links, archived_link))
-        return (1, 0)
+        return 1, 0
     else:
         print("[{:02d}/{}]: <NOW> {}".format(ind, len_links, archived_link))
-        return (0, 1)
+        return 0, 1
 
 
-def repl(args):
-    finish_words = ["end", "exit", "exit()", "break", "bye", ":q", "finish"]
+def wbsv_repl(args: argparse.Namespace) -> None:
+    finish_words = {"end", "exit", "exit()", "break", "bye", ":q", "finish"}
```

Comment:
```suggestion
    finish_words = ("end", "exit", "exit()", "break", "bye", ":q", "finish")
```

Reply:
I used a set for constant time lookup, could you explain why it should be changed to a tuple?